packages/account/src/Components/Routes/binary-routes.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+import { PlatformContext } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import getRoutesConfig from 'Constants/routes-config';
+import RouteWithSubRoutes from './route-with-sub-routes';
+import { TPlatformContext } from 'Types';
+
+type TBinaryRoutesProps = {
+    is_logged_in: boolean;
+    is_logging_in: boolean;
+};
+
+const BinaryRoutes = (props: TBinaryRoutesProps) => {
+    const { is_appstore } = React.useContext<TPlatformContext>(PlatformContext);
+
+    return (
+        <React.Suspense
+            fallback={
+                <div>
+                    <Localize i18n_default_text='Loading...' />
+                </div>
+            }
+        >
+            <Switch>
+                {getRoutesConfig({ is_appstore }).map((route, idx: number) => (
```

Comment:
I believe react knows that index is a number. BTW, is there any way to avoid using index in map here?

Reply:
There is no prop from route that can be relied upon. Hence going with index